source/api_documentation/network_integration/js_tags/_get_js_tag.rst

```diff
@@ -0,0 +1,156 @@
+
+
+.. container:: endpoint-long-description
+
+  .. rubric:: Examples
+
+  Read an Invoca Tag
+
+  Endpoint:
+
+  ``https://invoca.net/api/@@NETWORK_API_VERSION/<network_id>/js_tags/<js_tag_id>.json``
+
+  Response Code: 200
+
+  Response Body:
+
+  .. warning::
+      The provided documentation and JSON response for JS Tag Revision documentation might change in the next release.
```

Comment:
Added these warnings throughout as the JS Tag Revision tickets https://invoca.atlassian.net/browse/STORY-16628 and https://invoca.atlassian.net/browse/STORY-16629 will add new API version and that wouldn't have as many details.